### DISCUSSION TITLE
Can buildah run on a Kubernetes EKS if using Bottlerocket as container host?

### DISCUSSION THREAD

**Discussion by ecerulm** [containers/buildah#5088] [Q&A]
The [documentation][1] for AWS Bottlerocket says 
> Bottlerocket does not currently support user namespaces. 

when I run buildah on EKS I get 

```
buildah
Error during unshare(CLONE_NEWUSER): No space left on device
User namespaces are not enabled in /proc/sys/user/max_user_namespaces.
cat /proc/sys/user/max_user_namespaces
0
```

Does this means that buildah can't be used at all with Bottlerocket hosts?

[1]: https://github.com/bottlerocket-os/bottlerocket/blob/develop/SECURITY_GUIDANCE.md#do-not-run-containers-as-uid-0

**Comment by rhatdan** [comment]
Buildah could be run within a privileged POD.  It needs multiple UIDs and either CAP_SETUID,CAP_SETGID or CAP_SYS_ADMIN.
